addons/website_sale_comparison/views/website_sale_comparison_view.xml

```diff
@@ -18,11 +18,19 @@
         <field name="type">ir.actions.act_window</field>
         <field name="res_model">product.attribute.category</field>
         <field name="view_mode">tree</field>
+        <field name="help" type="html">
+            <p class="o_view_nocontent_smiling_face">
+                Create a new attribute category
+            </p><p>
```

Comment:
For what its worth, it seems like every single occurrence of double `<p/>` in `help` field have this weird closing and opening tag on the same line.
Didn't check if that's really needed (might be due to tab/newline glitch otherwise I guess), but anyway it's consistent with the rest of the codebase